tdrs-backend/tdpservice/parsers/schema_defs/tanf/t1.py

```diff
@@ -7,258 +7,302 @@
+                    result_function=validators.oneOf((1, 2)),
+                ),
+                validators.if_then_validator(
+                    condition_field="SANC_REDUCTION_AMT",
+                    condition_function=validators.isLargerThan(0),
+                    result_field="SANC_TEEN_PARENT",
+                    result_function=validators.oneOf((1, 2)),
+                ),
+                validators.if_then_validator(
+                    condition_field="SANC_REDUCTION_AMT",
+                    condition_function=validators.isLargerThan(0),
+                    result_field="NON_COOPERATION_CSE",
+                    result_function=validators.oneOf((1, 2)),
+                ),
+                validators.if_then_validator(
+                    condition_field="SANC_REDUCTION_AMT",
+                    condition_function=validators.isLargerThan(0),
+                    result_field="FAILURE_TO_COMPLY",
+                    result_function=validators.oneOf((1, 2)),
+                ),
+                validators.if_then_validator(
+                    condition_field="SANC_REDUCTION_AMT",
+                    condition_function=validators.isLargerThan(0),
+                    result_field="OTHER_SANCTION",
+                    result_function=validators.oneOf((1, 2)),
+                ),
+                validators.if_then_validator(
+                    condition_field="OTHER_TOTAL_REDUCTIONS",
+                    condition_function=validators.isLargerThan(0),
+                    result_field="FAMILY_CAP",
+                    result_function=validators.oneOf((1, 2)),
+                ),
+                validators.if_then_validator(
+                    condition_field="OTHER_TOTAL_REDUCTIONS",
+                    condition_function=validators.isLargerThan(0),
+                    result_field="REDUCTIONS_ON_RECEIPTS",
+                    result_function=validators.oneOf((1, 2)),
+                ),
+                validators.if_then_validator(
+                    condition_field="OTHER_TOTAL_REDUCTIONS",
+                    condition_function=validators.isLargerThan(0),
+                    result_field="OTHER_NON_SANCTION",
+                    result_function=validators.oneOf((1, 2)),
+                ),
+                validators.sumIsLarger(
+                    (
+                        "AMT_FOOD_STAMP_ASSISTANCE",
+                        "AMT_SUB_CC",
+                        "CASH_AMOUNT",
+                        "CC_AMOUNT",
+                        "TRANSP_AMOUNT",
+                        "TRANSITION_SERVICES_AMOUNT",
+                        "OTHER_AMOUNT",
+                    ),
+                    0,
+                ),
             ],
             fields=[
                   Field(item="0", name='RecordType', type='string', startIndex=0, endIndex=2,
-                        required=True, validators=[]),
+                        friendly_name="record type", required=True, validators=[]),
```

Comment:
just a general question - should the formatting be all lower-case or capitalized for the friendly name (`Record type` or `Record type`)? I could see wanting to use different formatting in different situations, so perhaps this could be done with a python util

Reply:
The ticket needed lower case since names are being replaced with friendly names in the error message.

Reply:
also is this true for acronyms? There are times when you have all caps and sometimes all lowercase.

e.g. SSI and ssi 

I think we should make them all the same.